services/tasks.py

```diff
@@ -95,7 +126,18 @@ def run(self, service_id, **kwargs):
             status = self.get_health(service.url, service.token)
             try:
                 result = status.json()
+
+                if (not service.up and result['up'] and service.last_up_at):
+                    downtime = timezone.now() - service.last_up_at
+                    fire_metric.apply_async(kwargs={
+                        "metric_name": 'services.downtime.%s.sum' % (
+                            utils.normalise_string(service.name)),
+                        "metric_value": downtime.seconds // 60
```

Comment:
I'm only firing the metric if it changes from down to up, then using the last time it was up until current time to see how long it was down.

Are you saying fire the metric on every health check not just if it changes?